packages/web/components/alert/prettify.ts

```diff
@@ -39,11 +46,32 @@ export function prettifyTxError(
       if (matchSignatureVerificationFailed.length >= 3) {
         const sequence = matchSignatureVerificationFailed[2];
         if (!Number.isNaN(parseInt(sequence))) {
-          return `You have too many concurrent txs going on! Try resending after your prior tx lands on chain. (We couldn't send the tx with sequence number ${sequence})`;
+          return ["errors.sequenceNumber", { sequence }];
         }
       }
     }
 
+    // Failed swap due to slippage
+    const matchFailedSwapSlippage = message.match(regexFailedSwapSlippage);
+    if (matchFailedSwapSlippage) {
+      if (matchFailedSwapSlippage.length >= 2) {
+        const denom = matchFailedSwapSlippage[1];
```

Comment:
how do we know it will always be the [1] index?

Reply:
I think it's reasonable to expect it will be 1 since the chain errors are highly deterministic, and the regex is specific enough to assume it will only match that error case coming from chain